src/components/templates/DefaultForm/index.tsx

```diff
@@ -19,8 +19,27 @@ export default function DefaultForm(props) {
   let hasSingleChildWhichIsReference = false;
   const instructionText = props.instructions === 'none' ? '' : props.instructions;
   const instructionExists = instructionText !== undefined && instructionText !== '';
+
+  const settingTargetForAnchorTag = () => {
+    const instructionDiv = document.getElementById('instructions');
+    const keyText = '(opens in same tab)';
```

Comment:
use the translation text, example in line 43 of this file

Reply:
Updated as required